### DISCUSSION TITLE
Is there an example of a project that setup GoReleaser with GOEXPERIMENT=boringcrypto?

### DISCUSSION THREAD

**Discussion by linouk23** [goreleaser/goreleaser#3832] [Q&A] [answered]
When I tried to add `GOEXPERIMENT=boringcrypto` (and `CGO_ENABLED=1`) I run into 
```
runtime/cgo
gcc: error: unrecognized command line option '-marm'
```

during `go release` step.

References:
* https://kupczynski.info/posts/fips-golang/
* https://developers.redhat.com/articles/2022/05/31/your-go-application-fips-compliant#

So I was wondering if there's a working example for that.

Thanks!

**Comment by caarlos0** [comment, marked as answer]
Not that I'm aware of, but I would guess it might be a matter of gcc versions...

PS; if you use CGO, you'll need either goreleaser-pro and split/merge builds, or goreleaser-cross, or maybe try with zig.
